src/blocks/video.js

```diff
@@ -9,11 +9,11 @@ module.exports = Block.extend({
   // more providers at https://gist.github.com/jeffling/a9629ae28e076785a14f
   providers: {
     vimeo: {
-      regex: /(?:http[s]?:\/\/)?(?:www.)?vimeo.com\/(.+)/,
+      regex: /(?:http[s]?:\/\/)?(?:www.)?vimeo\.co(?:.+(?:\/)([^\/].*)+$)/,
```

Comment:
Not a problem. 

Formats tested against goes as follows.

https://vimeo.com/channels/staffpicks/131417856

https://vimeo.com/groups/weekendchallenge/videos/131493293

https://vimeo.com/17853140

Unfortunately It won't take the on-demand video formats but that's because they don't have a unique ID string to work with. But I'd say it not a significant issue since the use case would be quite small and it wasn't something it was capable of before.